community/index/src/test/java/org/neo4j/index/gbptree/InternalTreeLogicTest.java

```diff
@@ -369,23 +426,25 @@ public void modifierMustRemoveFromLeftChild() throws Exception
     public void modifierMustRemoveFromRightChildButNotFromInternalWithHitOnInternalSearch() throws Exception
     {
         // given
-        node.initializeLeaf( cursor, STABLE_GENERATION, UNSTABLE_GENERATION );
+        node.initializeLeaf( cursor, stableGen, unstableGen );
         for ( int i = 0; !structurePropagation.hasSplit; i++ )
         {
             insert( i, i );
         }
         long rootId = newRootFromSplit( structurePropagation );
 
-        // when key to remove exists in internal
+        // when and key to remove exists in internal
```

Comment:
Probably a comment moved up from a lower part of the test and just put a when in front. Removing it.